### ISSUE TITLE
Error spamming when eslint plugin not found

### ISSUE THREAD

**Issue by aaronjensen** [AtomLinter/linter-eslint#189]
Sometimes I open a file from w/in another directory of my project such as within my node_modules directory. If that file has a corresponding .eslintrc, and that .eslintrc refers to a plugin that is not installed (like `eslint-config-airbnb`), I get an error:

```
Error: Cannot find module 'eslint-config-airbnb' Referenced from: /Users/aaronjensen/Source/project/node_modules/redux/.eslintrc
```

This isn't terrible. What is though, is that every change I make to the file pops another error over and over again. I have to disable linter-eslint in order for my editor to be usable.

It'd be nice if that error showed up in the bottom panel, like a lint error, and didn't keep coming up. Or, if it showed up once per file and never again.  Thanks!

**Comment by Nate-Wilkins** [comment]
This. :+1:

**Comment by pitaj** [comment]
Why not just remove it from your `.eslintrc`?

I'm not excusing the warning spam, but I'm just wondering.

**Comment by aaronjensen** [comment]
@pitaj they're usually in `node_modules`, so they should be ignored.

> In addition to any patterns in a .eslintignore file, ESLint always ignores files in node_modules/**.

**Comment by pitaj** [comment]
Oh. Your problem is the same as #192

**Comment by steelbrain** [comment]
Fixed by v5.0.0
